dfusion_rust_core/src/models/order.rs

```diff
@@ -44,6 +91,21 @@ impl From<mongodb::ordered::OrderedDocument> for Order {
     }
 }
 
+impl Into<Entity> for Order {
+    fn into(self) -> Entity {
+        let batch_info = self.batch_information.unwrap();
```

Comment:
Maybe just set the fields if the value is some? Right now, we crash if the value is not there.